arbor/fvm_lowered_cell_impl.hpp

```diff
@@ -771,7 +805,11 @@ void fvm_lowered_cell_impl<Backend>::resolve_probe_address(
         cable_probe_ion_diff_concentration,
         cable_probe_ion_diff_concentration_cell,
         cable_probe_ion_ext_concentration,
-        cable_probe_ion_ext_concentration_cell>;
+        cable_probe_ion_ext_concentration_cell,
+        cable_probe_density_prng_state,
```

Comment:
no I am not sure :) I want to test the state, and this is why I added the probes... Any idea how not to leak them to the users?

Reply:
Poke the binary data in shared state from the test suite? Should be possible -- if potentially brittle -- by using these 
private field access macros.

Maybe it's better to hide these probes behind a new feature flag, eg `ARB_DEBUG` or `ARB_INTERNAL`. These
could then also be used to expose other internals to the tests instead of these macros? I'd ask you to do it for
the PRNG state only, since once we release that into the wild _someone_ will depend on it and write out an issue
for the rest.

Reply:
I removed the probes